pages/create.php

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<html lang="en">
+<?php
+error_reporting(-1);
+ini_set('display_errors', 1);
+require_once '../config/connect.php';
+require_once '../blocks/header.php';
+
+$lastname = $_POST['lastname'];
```

Comment:
You definitely need to check that this data exist in $_POST before you will do something like sql insert